client/js/commands/index.ts

```diff
@@ -1,19 +1,11 @@
-// Taken from views/index.js
-
-// This creates a version of `require()` in the context of the current
-// directory, so we iterate over its content, which is a map statically built by
-// Webpack.
-// Second argument says it's recursive, third makes sure we only load javascript.
-const commands = require.context("./", true, /\.ts$/);
-
-export default commands.keys().reduce<Record<string, unknown>>((acc, path) => {
-	const command = path.substring(2, path.length - 3);
-
-	if (command === "index") {
-		return acc;
-	}
-
-	acc[command] = commands(path).default;
-
-	return acc;
-}, {});
+import {input as collapse} from "./collapse";
+import {input as expand} from "./expand";
+import {input as join} from "./join";
+import {input as search} from "./search";
+
+export const commands = {
+	collapse: collapse,
+	expand: expand,
+	join: join,
+	search: search,
+};
```

Comment:
This we probably also need to do with the other things that do the dynamic import...
While it does safe some typing, this is preventing us from moving to modules (we need top level imports or wait for the promise if we do it dynamically), is bad for tree shaking and simply put not needed as all our stuff is available at build time, we don't have plugins or such